main.go

```diff
@@ -574,3 +572,14 @@ func whenForced(forced, unforced string) string {
 	}
 	return unforced
 }
+
+func browserCmd() string {
+	switch runtime.GOOS {
+	case "darwin":
+		return "/usr/bin/open"
+	case "windows":
+		return "cmd /c start"
```

Comment:
Does this work? Won't this look for an executable named "cmd /c start" rather than an executable with name "cmd" and args ["/c", "start"]`?

Reply:
Yikes, good catch. Should be fixed now.